test/tests/variables/optionally_coupled/tests

```diff
@@ -47,4 +47,24 @@
     design = 'Coupleable.md'
     issues = '#11920'
   [../]
+
+  [./catch_out_of_bound_default_access_coupledValue]
+    type = 'RunException'
+    input = 'optionally_coupled.i'
+    cli_args = 'Kernels/wrong_index/type=CoupledForceWrongIndex Kernels/wrong_index/variable=u'
+    expect_err = 'MOOSE does not currently support a coupled vector where some parameters are reals and others are variables'
+    requirement = "Requested component 2 of coupled value v is out of range."
+    design = 'Coupleable.md'
+    issues = '#11920'
+  [../]
+
+  [./catch_out_of_bound_default_access_coupled]
+    type = 'RunException'
+    input = 'optionally_coupled.i'
+    cli_args = 'Kernels/wrong_index/type=CoupledForceWrongIndex Kernels/wrong_index/variable=u Kernels/wrong_index/access_value=false'
+    expect_err = 'MOOSE does not currently support a coupled vector where some parameters are reals and others are variables'
+    requirement = "Requested component 2 of coupled value v is out of range."
+    design = 'Coupleable.md'
+    issues = '#11920'
+  [../]
```

Comment:
Your tests are not testing the error message you added. I think you are hitting the same check as in your 2 tests above.

Reply:
I guess I copy pasted things to the wrong fields. :)